src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,202 @@
+    Given this school exists:
+            | schoolId     | nameOfInstitution                    | gradeLevels                   | educationOrganizationCategories      |
+            | 5            | School with max edorgId value        | [ "Postsecondary" ]           | [ "School" ]                         |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=ut+austin+college+of+education+graduate" 
+            """
+            """
+    Then it should respond with 200
+    And "nameOfInstitution: UT Austin College of Education Under Graduate" 
+
+  @ignore
+  Scenario: Ensure clients can GET information changing the casing of the query value to be all uppercase
+    Given a school exists:
+            | schoolId     | nameOfInstitution                                    | gradeLevels               | educationOrganizationCategories      |
+            | 6            | UT Austin College of Education Under Graduate        | [ "Postsecondary" ]       | [ "Educator Preparation Provider" ]  |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=UT+AUSTIN+COLLEGE+OF+EDUCATION+UNDER+GRADUATE" 
+            """
+            """
+    Then it should respond with 200
+    And "nameOfInstitution: UT Austin College of Education Under Graduate" 
+
+
+  @ignore
+  Scenario: Ensure clients can GET information filtering by totalCount
+    Given 8 schools exist:
+    When sending a GET request to "/ed-fi/schools?totalCount=true&limit=7" 
+    ##And "totalCount" 
+            """
+            """
+    Then getting less than the "totalCount"
+    And "totalCount: 7"
+
+  
+  @ignore
+  Scenario: Ensure clients can't GET information when filtering using invalid values
+    When sending a GET request to "/ed-fi/schools?limit=-1" using values as
+            | Values                       |
+            | -1                           |
+            | 'zero'                       |
+            | '5; select * from users'     |
+            | '0)'                         |
+            | '1%27'                       |   
+    Then it should respond with 400
+    And response is
+             """
+              {
+                "detail": "Data validation failed. See 'validationErrors' for details.",
+                "type": "urn:ed-fi:api:bad-request:data",
+                "title": "Data Validation Failed",
+                "status": 400,
+                "correlationId": "9e4f71dd-013c-41ee-9eaf-9c3dc368d206",
+                "validationErrors": {
+                  "$.Limit": [
+                    "The value 'zero' is not valid for Limit."
+                  ]
+                }
+              }
+             """ 
+  
+  @ignore
+  Scenario: Ensure clients can get information when filtering by limit and and a valid offset 
+    Given 5 schools exist:
+            | schoolId     | nameOfInstitution                               | 
```

Comment:
I think this table also need `gradeLevels` and the categories, so that it matches what we had in another feature file. Otherwise, it will be confusing in the C# step definition, because we'll have to deal with two different table layouts.

Reply:
Done